pyfr/solvers/baseadvecdiff/system.py

```diff
@@ -15,16 +15,33 @@ def deps(dk, *names): return self._kdeps(k, dk, *names)
         g1 = self.backend.graph()
         g1.add_mpi_reqs(m['scal_fpts_recv'])
 
+        # Perform post-processing of the previous solution stage
+        g1.add_all(k['eles/filter_solution'])
```

Comment:
Rename kernel so that it less confusing with the modal filter?  (Or rename the modal filter kernel?)

Reply:
Renamed both